File: javascript/1045.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var values = input.split(' ');
var a = parseFloat(values[0]);
var b = parseFloat(values[1]);
var c = parseFloat(values[2]);
var values = [a, b, c];
var sorting = values.sort((a, b) => {return b - a})
a = sorting[0]
b = sorting[1]
c = sorting[2]
if(a >= b+c) {
     console.log('NAO FORMA TRIANGULO');
} else {
     if(a**2 == b**2 + c**2) {
          console.log('TRIANGULO RETANGULO');
     }
     if(a**2 > b**2 + c**2) {
          console.log('TRIANGULO OBTUSANGULO');
     }
     if(a**2 < b**2 + c**2) {
          console.log('TRIANGULO ACUTANGULO');
     }
     if(a === b && b === c) {
          console.log('TRIANGULO EQUILATERO');
     }
     if((a === b && a !== c) || (a === c && a !== b) || (b === c && b !== a)) {
          console.log('TRIANGULO ISOSCELES');
     }
}

File: python/1045.py
i=input().split(' ')
inputs=[float(i[0]),float(i[1]),float(i[2])]
inputs.sort()
a=inputs[2]
b=inputs[1]
c=inputs[0]

if a>=(b+c):
    print("NAO FORMA TRIANGULO")
else:
    if a**2 == (b**2+c**2):
        print("TRIANGULO RETANGULO")
    if a**2 > (b**2+c**2):
        print("TRIANGULO OBTUSANGULO")
    if a**2 < (b**2+c**2):
        print("TRIANGULO ACUTANGULO")
    if a==b and b==c:
        print("TRIANGULO EQUILATERO")
    if ((a == b and a != c) or (a == c and a!= b) or (b == c and b != a)):
        print("TRIANGULO ISOSCELES")
